### ISSUE TITLE
Ability to trigger script on successful extraction or change

### ISSUE THREAD

**Issue by Nebukadneza** [ldez/traefik-certs-dumper#4]
Hey,

thanks for this useful project. One thing that would be tremendously useful in automating certificate distribution in docker/docker-compose hosted environments is the ability to trigger some external thing (a small script, restarting a container, …) either on successful extraction, or even only if the extracted certificate changed. I realize the latter might be a bit out of scope for a mere dumper (in the classical unix philosophy). That case is also fine: a user-supplied script could run the check on content-change, and only restart/trigger/… its target service when needed.

Best & Thanks,
-Dario

**Comment by BeastleeUK** [comment]
I'd second this.  I've built my first docker image based FROM the docker image you created @ldez,.  I added an MD5 check on one of the certificates as part of a further script which then splist and convert the certificate.crt and converts to PFX for my Plex server.  I did use the --watch switch but this only re-dumps the certs, requiring a further monitoring process.  It would be ideal to have a --trigger or similar that can point to a script on any change.

Thanks for your hard work
Lee
